kbase-extension/static/kbase/js/widgets/function_output/kbaseFeatureSet.js

```diff
@@ -155,10 +156,14 @@ define (
                                 // every feature we get back here is something in the list
                                 for(var f=0; f<g.features.length; f++) {
                                     var aliases = "None";
-                                    if(g.features[f].aliases) {
-                                        if(g.features[f].aliases.length>0) {
-                                            aliases= g.features[f].aliases.join(', ');
+                                    if (g.features[f].aliases && g.features[f].aliases.length>0){
+                                        if (g.features[f].aliases[0] instanceof Array){
+                                            g.features[f].aliases = g.features[f].aliases.map(x => x[1]);
```

Comment:
It's the `x => x[1]` part that's ES6 syntax that may not parse in all (or any?) browsers. do you have an example narrative that I can test against?

Reply:
One of the feature sets here: https://ci.kbase.us/narrative/ws.28853.obj.1

Reply:
Arrow functions look like they are supported by all but IE11

Reply:
...which we ostensibly support. Also it's failing tests. Those probably need updated to handle arrow functions.

Reply:
There's some shim stuff I'd like to start looking in to using, since I want to get things moved all the way over to ES6. But that takes more time than we usually budget for. :(

Anywhere, there's this if you want to tinker, otherwise change it back to the ugly function(x) { return x[1]; } for now.
https://www.npmjs.com/package/karma-es6-shim

Reply:
Yeah, I won't claim to be knowledgeable enough to integrate in the shim so I'll just use the older syntax